src/fprime/util/commands.py

```diff
@@ -182,3 +184,47 @@ def run_code_format(
     for filename in parsed.files:
         clang_formatter.stage_file(Path(filename))
     return clang_formatter.execute(build, parsed.path, ({}, parsed.pass_through))
+
+
+def run_version_check(
+    base: Build, parsed: argparse.Namespace, _: Dict[str, str], __: Dict[str, str], ___
+):
+    """Print out versions to help debugging"""
+
+    print(
+        f"Python version: {sys.version_info.major}{sys.version_info.minor}.{sys.version_info.micro}"
```

Comment:
Add a "." between major and minor version,

Reply:
Ah! I'm so sorry!